i18n/zh/docusaurus-plugin-content-docs/current/architecture/edge/metamanager.md

```diff
@@ -27,42 +27,33 @@ metamanager then sends an asynchronous message to edged. edged processes the ins
 by starting the pod and populates the response in the message.
 metamanager inspects the message, extracts the response and sends it back to edged
 which sends it back to the cloud.
```

Comment:
need delete the english part

Reply:
Thank you for your advice, I have made the corresponding changes.